extensions/opentelemetry/opentelemetry-async-mutiny/runtime/src/main/java/io/quarkus/opentelemetry/async/mutiny/runtime/tracing/TracingMultiSubscriber.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.opentelemetry.async.mutiny.runtime.tracing;
+
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.smallrye.mutiny.subscription.MultiSubscriber;
+
+public class TracingMultiSubscriber<T> implements MultiSubscriber<T> {
+
+    private final Context context;
+
+    private final Subscriber<? super T> subscriber;
+
+    public TracingMultiSubscriber(final Context context, final Subscriber<? super T> subscriber) {
+        this.context = context;
```

Comment:
This is the assembly time context. Most probably you need the subscription time one (onSubscribe()).